src/server/game/Entities/DynamicObject/DynamicObject.cpp

```diff
@@ -135,7 +135,7 @@ void DynamicObject::Update(uint32 p_time)
             _aura->UpdateOwner(p_time, this);
 
         // _aura may be set to null in Aura::UpdateOwner call
-        if (_aura && (_aura->IsRemoved() || _aura->IsExpired()))
+        if ((_aura->IsRemoved() || _aura->IsExpired()))
```

Comment:
are we sure about this? it doesn't complain if **_aura** is null while it checks its method?

Reply:
Yes, because this this block is inside of an if (_aura) statement: https://github.com/azerothcore/azerothcore-wotlk/blob/bb4dc311593802e7fdf73a7aa6470da7ab9360d4/src/server/game/Entities/DynamicObject/DynamicObject.cpp#L132

Reply:
Oh damn, i didn't saw the UpdateOwner call - You're right with this one. Good catch!